### DISCUSSION TITLE
foodpanda斥資6億擴編團隊 祭高薪搶百名科技人才

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#461] [General]
foodpanda斥資6億擴編團隊 祭高薪搶百名科技人才
Foodpanda的勢力越來越壯大了
https://www.technice.com.tw/cloudtech/189/

**Comment by BingRong1217** [comment]
現在越來越多人使用foodpanda此種外送平台，越加茁壯也在預料之中

**Comment by boson13579** [comment]
因為越來越多人使用foodpanda平台，完善及擴充平台的功能也顯得極為重要，搞不好之後的客服也會交由AI管控，不僅為foodpanda省去更多人力成本，也避免客服人員會跟消費者起衝突的風險。

**Comment by larry40305** [comment]
之後為了減少人力成本，說不定就會引進AI機器人配送餐點。

**Comment by HsuChen88** [comment]
我只知道food panda廣告越打越兇，在廣告上所花的經費應該也蠻多。

**Comment by xCk27x** [comment]
但我對這種外送平台是否需要提供更多使用者服務感到懷疑?
畢竟就算使用者體驗再好 大多數人還是只會看到折扣跟運費這兩個面向
感覺是要以拓展業務範圍跟地區當作主要目標才對

**Comment by Aerofe** [comment]
現在越來越多人從事科技業，也會導致科技業的人才逐漸飽和，之後想要在科技業拿不錯的薪資勢必要一直不斷的精進自己才不會被別人擠下

**Comment by Valentinaa0214** [comment]
越來越多產業與科技做跨領域結合了，各行各業都需要科技做結合加已精進，希望外送平台也能夠借助這些科技力量，有效提升外送產業鏈運作效率，減少資源浪費，落實企業環境永續責任。

**Comment by Raywei0214** [comment]
現在什麼都馬要結合ai ，沒有ai 就落伍了，有人提到說用ai機器人配送餐點 這個議題我已經在另一個貼文討論過了，在此不討論；但我覺得增加ai 的好處是可以方便管理系統，類似刀劍神域中，那個外掛小蘿莉 結衣，她原本的設定就是管理虛擬世界中，每個人的情感參數，類似這種ai 自動管理可以大幅減少系統debug 所需的人力成本以及時間，我覺得以後大部分產業都會朝這個方向發展。

**Comment by Vincent20030314** [comment]
外送和AI都是現在都是熱門產業，加在一起確實對能造福社會，但是這也代表現在工作機會更傾向科技產業。

**Comment by titan92601** [comment]
像這種外送產業一旦和AI一起發展起來效果其實非常恐怖的。Amazon 想必大家都略知一二，線上物流公司數一數二有名的，賺得錢可數都數不完。如今他們已經發展出來外送無人機，一旦這個東西普及的話，有機會改變全球經濟和市場呢！
